src/newSeed/stage4.ts

```diff
@@ -71,15 +71,21 @@ export class Stage4 extends BaseStage {
       message = "Please select a Start Date";
     } else if (!this.startTime) {
       message = "Please enter a value for the Start Time";
-    } else if (!(Number.parseInt(this.startTime.split(":")[0]) < 25)
-      || !(Number.parseInt(this.startTime.split(":")[1]) < 61)) {
+    } else if (!(Number.parseInt(this.startTime.split(":")[0]) >= 0)
```

Comment:
what happens if there is no ':'?

or what happens if the value is not a number (parseInt returns NaN)

Reply:
Validation flags it

Reply:
Validation catches all corner cases I tested but please in case you think of any other